vshard/util.lua

```diff
@@ -236,6 +236,19 @@ local function table_minus_yield(dst, src, interval)
     return dst
 end
 
+--
+-- Append all items of src to the end of dst. It is assumed both tables are
+-- arrays.
+--
+local function table_concat(dst, src)
```

Comment:
There is default `table.concat` this name confuses me a bit. Maybe table_ extend (as in python)?
https://docs.python.org/3/tutorial/datastructures.html#more-on-lists

Reply:
Renamed to `table_extend`.